lib/preview_web/router.ex

```diff
@@ -20,8 +20,8 @@ defmodule PreviewWeb.Router do
     pipe_through :browser
 
     live "/", SearchLive, :index
-    live "/preview/:package/:version/:filename", PreviewLive, :index
     live "/preview/:package/:version", PreviewLive, :index
+    live "/preview/:package/:version/show/*filename", PreviewLive, :index
```

Comment:
another nice benefit of `/show/` being part of the URL is we could later add `/raw/`.